functions.sh

```diff
@@ -95,11 +95,19 @@ download_iso() {
 }
 
 mount_iso() {
-	if findmnt "$ISOMNT" > /dev/null
+	LOOPDEV=$(/sbin/losetup -n -O NAME -j "$ISOPATH/$ISOFILE")
+	if [[ -n "$LOOPDEV" ]]
 	then
+		ISOMNT="$LOOPDEV"
 		umount_iso
```

Comment:
I don't know if it's needed to umount the iso first if ISOMNT is not hardcoded, but it looks good to me.